Sources/SVGLoader.swift

```diff
@@ -32,43 +32,78 @@ public final class SVGLoader {
             }
             """)
         }
+        
+        public func buildHtml(svg: String) -> String {
```

Comment:
I felt strange that this method is in a Style struct.
What do you think of moving this method to an SVG Loader member function?

Reply:
@kzumu I fixed it, could you check it?